client/me/purchases/manage-purchase/change-payment-method/index.jsx

```diff
@@ -157,32 +164,41 @@ ChangePaymentMethod.propTypes = {
 };
 
 const wpcom = wp.undocumented();
-const wpcomGetStoredCards = () => wpcom.getStoredCards();
+const wpcomAssignPaymentMethod = ( subscriptionId, stored_details_id, fn ) =>
+	wpcom.assignPaymentMethod( subscriptionId, stored_details_id, fn );
 
-function ChangePaymentMethodList() {
-	const [ formSubmitting ] = useState( false );
-	const translate = useTranslate();
+function ChangePaymentMethodList( {
+	currentPaymentMethod,
+	purchase,
+	isDataLoading,
+	successCallback,
+} ) {
+	const currentlyAssignedPaymentMethodId = 'existingCard-' + currentPaymentMethod.stored_details_id; // TODO: make this work for paypal.
 
-	const { storedCards } = useStoredCards( wpcomGetStoredCards, false );
-	const { isStripeLoading, stripeLoadingError, stripeConfiguration, stripe } = useStripe();
+	const [ formSubmitting, setFormSubmitting ] = useState( false );
+	const translate = useTranslate();
+	const { isStripeLoading, stripeLoadingError } = useStripe();
+	const [ paymentMethods ] = useAssignablePaymentMethods();
```

Comment:
This doesn't need to be a destructured array. I think it was like this back when this hook returned more than one variable, but it can go back to returning one (array) variable.

Reply:
To be clear, I'm suggesting we change `useAssignablePaymentMethods` to return `paymentMethods` and then to change this line to be `const paymentMethods = ...`